src/main/groovy/org/jetbrains/intellij/IntelliJPlugin.groovy

```diff
@@ -351,8 +346,9 @@ class IntelliJPlugin implements Plugin<Project> {
             dependsOn(JavaPlugin.JAR_TASK_NAME)
 
             if (taskName == PREPARE_UI_TESTING_SANDBOX_TASK_NAME) {
-                from project.zipTree(project.configurations.robotServerPluginImplementation.files.find { it.name.endsWith("zip") })
-                into Utils.pluginsDir(extension.sandboxDirectory, "-uiTest")
+                dependsOn(DOWNLOAD_ROBOT_SERVER_PLUGIN_TASK_NAME)
+                def downloadTask = project.tasks.findByName(DOWNLOAD_ROBOT_SERVER_PLUGIN_TASK_NAME) as DownloadRobotRegistryPluginTask
```

Comment:
You can do this on the call site, no need to have it in the common code

Reply:
moved it to  `configureProjectAfterEvaluate` method